assets/js/urlBuilder.js

```diff
@@ -0,0 +1,13 @@
+const allUrlToBuild = document.getElementsByClassName('js-build-url');
```

Comment:
This file has a name in camel case, and assets/js/highlight_scroll.js is in snake case.

you need to choose a camp (snake or kebab)

---

I would name the var `urls`